test/unit/specs/components/AudioTrack/audio-track.spec.js

```diff
@@ -16,9 +17,39 @@ const useVueI18n = (vue) => {
     i18n,
   }
 }
+const useVuexStore = (vue) => {
```

Comment:
Nice abstraction! Hopefully we can create similar abstraction for all tests that use testing-library, a utility similar to `render` that we had previously.